src/OpenTelemetry.Exporter.InMemory/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Buffer scopes when exporting `LogRecord`
```

Comment:
Mention `InMemoryExporter`?

Reply:
I thought it was a bit redundant to have that given it is the CHANGELOG for InMemoryExporter but no prob I added it :smile: